ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -275,12 +278,13 @@ public QueryMetadata getQueryExecutionPlan(final Query query) throws Exception {
     }
   }
 
-  private Pair<String, Statement> buildSingleQueryAst(
+  public Pair<String, Statement> buildSingleQueryAst(
```

Comment:
Why this change?

Reply:
The other one was public for test purpose. I made this one private.